### DISCUSSION TITLE
How to get the measure unit of the iModel?

### DISCUSSION THREAD

**Discussion by Snailya** [iTwin/itwinjs-core#4372] [Q&A]
How to get the measure unit like what MicroStation shows of the iModel?
![image](https://user-images.githubusercontent.com/19658624/192466491-cd6f0ae7-cf53-48fd-b234-9d60c66383e0.png)

**Comment by kabentley** [comment]
the units for all coordinate data in iModels is meters.

**Comment by pmconne** [comment]
Like @kabentley said, all distances in an iModel are stored in meters.
If you want to change how they are displayed, you can switch between metric and English unit systems as [illustrated here](https://www.itwinjs.org/sandboxes/JoshSchifter/Change%20Unit%20System).
More complex formatting options are available if you need them.

Note there is no concept of "solids working area" in an iModel.
